File: MiniDOMWebCtrls/DMD/Sistema/CUsersModuleHandler.cs

namespace minidom.Forms
{

    // Handler del module Utenti
    public class CUsersModuleHandler
        : CBaseModuleHandler
    {
        public CUsersModuleHandler()
        {
        }

        public CUsersModuleHandler(Sistema.CModule module) : this()
        {
            SetModule(module);
        }

        public override object GetInternalItemById(int id)
        {
            // Return MyBase.GetItemById(id)
            return Sistema.Users.GetItemById(id);
        }

        public override Databases.DBObjectCursorBase CreateCursor()
        {
            var cursor = new Sistema.CUserCursor();
            cursor.Nominativo.SortOrder = Databases.SortEnum.SORT_ASC;
            return cursor;
        }
    }
}

File: MiniDOMWebCtrls/DMD/Sistema/CUsersModuleHandler.vb
Imports minidom
Imports minidom.Sistema
Imports minidom.Forms
Imports minidom.WebSite
Imports minidom.Databases
Imports minidom.Forms.Utils

Imports minidom.Anagrafica
Imports minidom.XML

Namespace Forms

    'Handler del module Utenti
    Public Class CUsersModuleHandler
        Inherits CBaseModuleHandler

        Public Sub New()
            
        End Sub

        Public Sub New(ByVal [module] As CModule)
            Me.New()
            Me.SetModule([module])
        End Sub

        Public Overrides Function GetInternalItemById(id As Integer) As Object
            'Return MyBase.GetItemById(id)
            Return Sistema.Users.GetItemById(id)
        End Function



        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Dim cursor As New CUserCursor
            cursor.Nominativo.SortOrder = SortEnum.SORT_ASC
            Return cursor
        End Function



    End Class


End Namespace